docs/security/users-and-teams/auditing.md

```diff
@@ -7,7 +7,7 @@ position: 30
 For team members to collaborate in the deployment of software, there needs to be trust and accountability. Octopus Deploy captures audit information whenever significant events happen in the system.
 
 :::hint
-The [Audit Retention functionality](#manage-archived-audit-events) was introduced in **Octopus 2022.3** and is being rolled out in stages for our Cloud customers first. It will be made available to Self-hosted customers in **Octopus 2022.4**.
+The [Audit Retention functionality](#archived-audit-events) was introduced in **Octopus 2022.3** and is being rolled out in stages for our Cloud customers first. It will be made available to Self-hosted customers in **Octopus 2022.4**.
```

Comment:
I renamed the below existing section `Manage archived audit logs` to just `Archived audit logs`, as it confuses the general archived audit log concept with the actual `Manage archived audit logs` page (which is used for viewing and downloading archives)